app/scripts/metamask-controller.test.js

```diff
@@ -2840,4 +2840,144 @@ describe('MetaMaskController', () => {
       expect(browserPolyfillMock.storage.session.set).not.toHaveBeenCalled();
     });
   });
+
+  describe('MetaMetrics & authentication dependencies', () => {
+    let metamaskController;
+    let mockPerformSignIn;
+    let mockPerformSignOut;
```

Comment:
Nit (cleanup) - We don't need mutating variables, instead we can just return them in our arrange function.

```ts
const arrangeControllerState = (stateOverride) => {
  // ...
  return {
    metamaskController,
    mockPerformSignIn,
    mockPerformSignOut,
  }
}

```